### PR TITLE
show client time when script and lesson edit pages are saved

### PR DESCRIPTION
**Pull request by davidsbailey**
Finishes [PLAT-668].

before (levelbuilder):
Screen Shot 2021-01-08 at 8 40 24 AM

Screen Shot 2021-01-08 at 8 40 30 AM

before (localhost):
Screen Shot 2021-01-08 at 9 16 28 AM
Screen Shot 2021-01-08 at 9 16 34 AM

after (localhost):
Screen Shot 2021-01-08 at 9 22 27 AM
Screen Shot 2021-01-08 at 9 22 32 AM

## Testing story

Manually verified that the last saved timestamp matches my local system clock time on the script edit page and lesson edit page.

# Reviewer Checklist:

- [ ] Tests provide adequate coverage
- [ ] Privacy and Security impacts have been assessed
- [ ] Code is well-commented
- [ ] New features are translatable or updates will not break translations
- [ ] Relevant documentation has been added or updated
- [ ] User impact is well-understood and desirable
- [ ] Pull Request is labeled appropriately
- [ ] Follow-up work items (including potential tech debt) are tracked and linked

[PLAT-668]: https://codedotorg.atlassian.net/browse/PLAT-668

### CODE REVIEW

**Comment by davidsbailey** [comment]
apologies, I sent this too soon. I will mention you both again when tests are passing.

**Comment by dmcavoy** [comment]
I added `updatedAt` to the summarize methods for Script and Lesson to make this work. I think this means we don't need that information in the summarize anymore.

It should be removed from the following places:

https://github.com/code-dot-org/code-dot-org/blob/staging/dashboard/app/models/script.rb#L1352
https://github.com/code-dot-org/code-dot-org/blob/staging/dashboard/app/models/lesson.rb#L308
https://github.com/code-dot-org/code-dot-org/blob/staging/dashboard/app/controllers/lessons_controller.rb#L37
https://github.com/code-dot-org/code-dot-org/blob/staging/apps/src/lib/levelbuilder/lesson-editor/LessonEditor.jsx#L92
https://github.com/code-dot-org/code-dot-org/blob/staging/apps/test/unit/lib/levelbuilder/script-editor/ScriptEditorTest.jsx#L197
https://github.com/code-dot-org/code-dot-org/blob/staging/apps/test/unit/lib/levelbuilder/script-editor/ScriptEditorTest.jsx#L271

If you want me to follow up with that work let me know.

**Comment by davidsbailey** [comment]
@dmcavoy @bethanyaconnor this is ready for review now. Dani, thanks for the pointers to updatedAt.  I just went and removed them in this PR.

File: apps/test/unit/lib/levelbuilder/lesson-editor/LessonEditorTest.js [hunk 1]
```diff
@@ -139,11 +143,15 @@ describe('LessonEditor', () => {
     expect(wrapper.find('.saveBar').find('FontAwesome').length).to.equal(1);
     expect(lessonEditor.state().isSaving).to.equal(true);
 
+    clock = sinon.useFakeTimers(new Date('2020-12-01'));
```
Comment (dmcavoy) [inline apps/test/unit/lib/levelbuilder/lesson-editor/LessonEditorTest.js:146]: Cool! TIL :)

**Review by dmcavoy — APPROVED** [review]
Thanks for following up and cleaning up all those places where updatedAt was!

**Pull request merged**